sos/plugins/openstack_tripleo.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2019 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class OpenStackTripleO(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    '''Some TripleO logs and files (OpenStack installer)
```

Comment:
Minor nit: PEP257 specifies the use of triple double quotes instead of triple single quotes. Normally I wouldn't even mention this, but #1914 has been opened to correct this elsewhere, so might as well nip it early.

Reply:
OK! didn't know about that one, will correct!